udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/OrganizationPublishingForm.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <div ref="containerRef">
+  <div class="fr-container" ref="containerRef">
+    <Stepper :steps="steps" :currentStep="currentStep"/>
     <Step1CreateOrJoinOrganization
       v-if="currentStep === 0"
       :steps="steps"
```

Comment:
Are the steps still necessary if the Stepper is outside of the component ?

Reply:
No, they are remove from the step components. I'll update the form